### ISSUE TITLE
[ko] /ko/releases/version-skew-policy/ is not displayed well

### ISSUE THREAD

**Issue by jihoon-seo** [kubernetes/website#28590]
**This is a Bug Report**

**Problem:**

https://kubernetes.io/ko/releases/version-skew-policy/ is not displayed well.
![image](https://user-images.githubusercontent.com/46767780/123204551-4f72d800-d4f3-11eb-9a82-40d5285a3812.png)

---

English version:
https://kubernetes.io/releases/version-skew-policy/
![image](https://user-images.githubusercontent.com/46767780/123204592-60234e00-d4f3-11eb-93bc-d027ed4f60e8.png)

**Proposed Solution:**

**Page to Update:**
https://kubernetes.io/ko/releases/version-skew-policy/

**Comment by seokho-son** [comment]
```
---
reviewers:
- sig-api-machinery
- sig-architecture
- sig-cli
- sig-cluster-lifecycle
- sig-node
- sig-release
title: Version Skew Policy
type: docs
description: >
  The maximum version skew supported between various Kubernetes components.
---

<!-- overview -->
This document describes the maximum version skew supported between various Kubernetes components.
Specific cluster deployment tools may place additional restrictions on version skew.
```

English document has `type: docs`

but, Korean doc doesn't have it :) I think this will resolve the problem.
https://github.com/kubernetes/website/blob/main/content/ko/releases/version-skew-policy.md

@jihoon-seo Could you help ? ^^

**Comment by jihoon-seo** [comment]
No problem!
/assign

**Comment by jihoon-seo** [comment]
This issue was resolved by the PR #28618.
